webapp/src/main/webapp/iaf/gui/js/main.js

```diff
@@ -42,7 +42,7 @@ $(document).ready(function () {
 	}
 
 	window.addEventListener("keydown", function(e) {
-		if(e.which == 70 && (e.ctrlKey || e.metaKey)) {
+		if(e.which == 70 && (e.ctrlKey || e.metaKey) && e.shiftKey) {
```

Comment:
De nieuwe shortcut ook weergeven bij de searchbar (bijvoorbeeld in de placeholder).